src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js

```diff
@@ -57,12 +58,12 @@ function KeyBasedAssetTransferIntro() {
 
   return (
     <ThemeProvider theme={darkTheme}>
-      <ContainerWithHeader
-        putContentInScrollView
-        headerProps={{ floating: true }}
-        backgroundColor={darkTheme.colors.basic090}
-      >
-        <Content>
+      <Container>
+        <HeaderBlock
+          navigation={navigation}
+          noPaddingTop
+        />
+        <Content showsVerticalScrollIndicator={false}>
```

Comment:
1. If using `Container` from `components/modern/Layout` you probably would want to use `Content` component from there as well as it handles bottom safe are for iPhone X correctly.

2. Not sure if we would want to hide scroll indicator there. 

3. Additionally in the current state the top safe area is applied to the scroll view, making the top padding of content weird. Using `Content` from `modern/Layout` hopefully should solve it. If not then please do some style override on for that.

![image](https://user-images.githubusercontent.com/6368606/121894990-0dda7400-cd20-11eb-9d28-109ed488b998.png)

Reply:
@mdjastrzebski Done. Point 2-  scroll indicator was not there previously, so implemented in such a way.

Reply:
IMO I would keep the scroll indicator in default mode as there seems to be no UI or UX reason to hide it.

Reply:
@IAmKio are you aware of any reasons why might want to hide scroll indicator on this screen?